### DISCUSSION TITLE
URGENT - Field BT-135 does not show up on TED or in generated PDF, but is present in XML from TED

### DISCUSSION THREAD

**Discussion by OliverStingl** [OP-TED/eForms-SDK#1058] [Q&A] [answered]
one of our clients published a form # 25 through our system and field BT-135 (justification) is missing in generated PDFs and is not showing up on the TED-website, but is very important for this procedure. The field is present in the XML provided by the TED-website.

https://ted.europa.eu/de/notice/-/detail/659079-2024

there's a deadline running

**Comment by pdonohoe** [comment, marked as answer]
There is a mistake in the XML. The text is not identifiable as BT-135, as the container element is missing field BT-136 Direct Award Justification Code. Without this missing field, the text could also be interpreted as BT-1351 Procedure Accelerated Justification.

**Reply by OliverStingl** [reply]
Thank you, the validation service did not mark the missing code as an issue. 
Is this intentional or is there an issue with this validation?

**Reply by pdonohoe** [reply]
A validation check which would have detected this issue was introduced with SDK 1.12. Such validation checks are not added to previous SDK versions as this would potentially make notices that have been already published, invalid.

**Reply by OliverStingl** [reply]
Thank you very much!

**Comment by bertrand-lorentz** [comment]
@OliverStingl : Please note that for problems related to the publication or display of specific notices, you should contact the TED helpdesk, in particular in case of urgency.
We cannot guarantee a timely response for issues or discussions on GitHub.
